### PR TITLE
Pop up size consistency and map panning to center the pop

### PR DESCRIPTION
**Pull request by Frankdevlopes**
-Replaced the closebutton with the reusable closebutton.
-Modified the seller.name on wrapping due to its length, the image reduces in size to avoid container height distortion.
-Centered the pop-up avoiding it from been overlayed by modifying handleclick function

### CODE REVIEW

**Review by DarinHajou — APPROVED** [review]
LGTM! Just resolve the conflict.

File: src/components/shared/map/MapMarkerPopup.tsx [hunk 1]
```diff
@@ -9,12 +10,22 @@ import logger from '../../../../logger.config.mjs';
 
 const MapMarkerPopup = ({ seller }: { seller: any }) => {
   const t = useTranslations();
-  const locale = useLocale();
```
Comment (swoocn) [inline src/components/shared/map/MapMarkerPopup.tsx:12]: Is there a reason `locale` is removed?  
This is needed to persist the preferred language back to the Buy From Seller screen.

File: src/components/shared/map/MapMarkerPopup.tsx [hunk 2]
```diff
@@ -9,12 +10,22 @@ import logger from '../../../../logger.config.mjs';
...
+      // Reduce image height slightly if name wraps (i.e., height > 22px)
+      setImageHeight(nameHeight > 22 ? 75 : 100);
+    }
+  }, [seller.name]);
 
   const imageUrl =
     seller.image && seller.image.trim() !== ''
       ? seller.image
-      : '/images/logo.svg';
+      : process.env.NEXT_PUBLIC_IMAGE_PLACEHOLDER_URL || '/images/shared/upload.png';
```
Comment (swoocn) [inline src/components/shared/map/MapMarkerPopup.tsx]: Another confusing adjustment that may be the result of an unsynced branch with the current dev..? 🤔

File: src/components/shared/map/MapMarkerPopup.tsx [hunk 3]
```diff
@@ -45,35 +65,45 @@ const MapMarkerPopup = ({ seller }: { seller: any }) => {
...
+          margin: '0 auto',
+          marginBottom: '8px',
+        }}
+      >
+        <TrustMeter ratings={seller.trust_meter_rating} />
       </div>
 
       {/* Link to Buy button */}
       <div style={{ display: 'flex', justifyContent: 'center', marginTop: '5px' }}>
-        <Link
```
Comment (swoocn) [inline src/components/shared/map/MapMarkerPopup.tsx:73]: We need to revert this change in order to persist preferred language.

File: src/components/shared/map/Map.tsx [hunk 4]
```diff
@@ -4,6 +4,7 @@ import React, { useEffect, useState, useCallback, useContext } from 'react';
 import { MapContainer, Marker, Popup, TileLayer, useMapEvents } from 'react-leaflet';
 import L, { LatLngExpression, LatLngBounds, LatLngTuple } from 'leaflet';
 import _ from 'lodash';
+import { CloseButton } from '../Forms/Buttons/Buttons';
```
Comment (swoocn) [inline src/components/shared/map/Map.tsx]: 👍

**Review by swoocn — APPROVED** [review]
LGTM @Frankdevlopes, great work! 😎👌
**Just a quick note** - in the future, please take a moment to review your changes before committing them.  It seems something happened with your branch that unintentionally removed some existing adjustments i.e., locale and image placeholder.  I’ve gone ahead and restored it.  Much appreciated!
